src/commonMain/kotlin/exchange.dydx.abacus/state/manager/V4StateManagerAdaptor.kt

```diff
@@ -717,6 +727,35 @@ class V4StateManagerAdaptor(
         }
     }
 
+    private fun delayedInRequestTime(networkState: NetworkState): Boolean {
```

Comment:
Feels like a bit of a hacky way to do this, using random numbers for the time duration. Can't we use the block numbers being returned from indexer and validator endpoints to determine this?